packages/block-library/src/index.js

```diff
@@ -330,3 +330,5 @@ export const __experimentalRegisterExperimentalCoreBlocks = process.env
 				.forEach( ( { init } ) => init() );
 	  }
 	: undefined;
+
+export { default as BlockKeyboardShortcuts } from './block-keyboard-shortcuts';
```

Comment:
Can we make it a private component for now. (until we make these transforms or something else)

Reply:
> Can we make it a private component for now. (until we make these transforms or something else)

@youknowriad thanks. Do you mean using lock/unlock? See last commit.

Reply:
yes, that was correct 👍